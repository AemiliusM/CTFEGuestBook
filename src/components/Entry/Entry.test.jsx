import React from 'react';
import { render } from '@testing-library/react';
import Entry from './Entry';

it('should render an entry', () => {
  const { container } = render(
    <Entry entry={{ name: 'billy', message: 'hello' }} />
  );
  expect(container).toMatchSnapshot();
});
