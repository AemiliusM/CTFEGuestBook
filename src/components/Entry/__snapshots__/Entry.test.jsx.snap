// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an entry 1`] = `
<div>
  <div>
    <h2>
      billy
    </h2>
    <p>
      hello
    </p>
  </div>
</div>
`;
